pages/common/content/success-messages.js

```diff
@@ -38,6 +38,18 @@ module.exports = {
     }
   },
   pil: {
+    revocation: {
+      resubmitted: {
```

Comment:
`resubmitted`?

Reply:
yeah, resubmitted === submitted when it comes to messaging - we have content for resubmitted and not submitted, it avoids repeating them all for submitted

Reply:
it should definitely be the other way round but it isn't, and it's used _everywhere_ and.. oh my fuck budget just ran out